tests/js/ternaries/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -892,8 +892,10 @@ a
 																),
 																a()
 																	? {
-																			a: a(),
-																			b: [],
+																			a:
+																				a(),
+																			b:
+																				[],
```

Comment:
These look broken?

Reply:
It's beyond `printWidth`. If the names of the keys were longer, it wouldn't look that bad. Special handling for short keys is a separate open issue: https://github.com/prettier/prettier/issues/3711